Dockerfile

```diff
@@ -1,23 +1,16 @@
 FROM python:3.6-alpine
 
-RUN apk --no-cache add \
-    python3 \
-    git \
-    libpq \
-    wget \
-    libffi \
-    libffi-dev \
-    ca-certificates \
-    python3-dev \
-    postgresql-dev \
-    build-base \
-    bash \
-    curl
-RUN update-ca-certificates
+RUN apk add --update --no-cache libpq postgresql-dev libffi libffi-dev bash libstdc++ nodejs
+RUN apk add --update --no-cache --virtual=build-dependencies build-base 
+RUN apk --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --update add leveldb leveldb-dev
 
 WORKDIR /app
 ADD requirements.txt .
 RUN pip install -r requirements.txt
+RUN npm install -g os-types
```

Comment:
What's this used for?

Reply:
This is the code that builds the FDP model - it's used in the packager as well.

Reply:
Is it used in the os-conductor codebase, I can't find it.

Reply:
It is used by datapackage-pipelines-fiscal, inside the model.py processor

Reply:
Can you add a comment here to say what's using it, and a note in datapackage-pipelines-fiscal's README file to state it as a dependency.

Reply:
will do.

Reply:
done + https://github.com/openspending/datapackage-pipelines-fiscal/commit/64bf315e53cec4fdf8f0a03737f5f63fd38f8fa4